src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -397,6 +350,23 @@ typedef struct {
     char kvsname[MPIDI_KVSAPPSTRLEN];
     char pname[MPI_MAX_PROCESSOR_NAME];
     int port_name_tag_mask[MPIR_MAX_CONTEXT_MASK];
+
+    /* Capability settings */
+#ifdef MPIDI_OFI_ENABLE_RUNTIME_CHECKS
```

Comment:
I think this goes back to what we were talking about on the phone. While it would be nice to be able to select some capabilities as static and others as runtime, there's no way to query that from OFI. You have to either specify every capability you want or none. If you only specify some, you probably won't get good output for the rest. Especially for the things that aren't "capabilities" like scalable endpoints.